### PR TITLE
updated readme file for cocoapods

### PR DESCRIPTION
**Pull request by kneekey23**
*Description of changes:*
Updated readme with cocoapod instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,11 +36,43 @@ Amplify requires Xcode 11 or higher to build.
...
+        $ gem install cocoapods
+        $ pod setup
+
+    Depending on your system settings, you may have to use `sudo` for installing `cocoapods` as follows:
+
+        $ sudo gem install cocoapods
+        $ pod setup
+2. In your project directory (the directory where your `*.xcodeproj` file is), type `pod init` and open the Podfile that was created. Add the `Amplify` pod and any plugins you would like to use. Below is an example of what a podfile might look like if you were going to use the Predictions plugin.
+
+        source 'https://github.com/CocoaPods/Specs.git'
```
Comment (palpatim) [inline README.md]: We should be using the CDN rather than the github spec.

File: README.md [hunk 2]
```diff
@@ -36,11 +36,43 @@ Amplify requires Xcode 11 or higher to build.
...
+        
+        platform :ios, '13.0'
```
Comment (palpatim) [inline README.md]: This platform restriction is only necessary for Predictions, or DataStore+Sync. Other customers can use 11.0

Comment (kneekey23) [reply README.md]: yes we should add another example not dependent on 13, will do in follow up PR

File: README.md (hunk 1 above)
Comment (kneekey23) [reply README.md]: will do in follow up PR

**Pull request merged**